test-data/unit/stubgen.test

```diff
@@ -20,42 +20,52 @@ def g(arg): ...
 def f(a, b=2): ...
 def g(b=-1, c=0): ...
 [out]
-def f(a, b: int = ...): ...
-def g(b: int = ..., c: int = ...): ...
+def f(a, b: int=...): ...
```

Comment:
ok, I was used to the old rule, not the special case for annotations. I'll change that